src/ContentValidationListenerFactory.php

```diff
@@ -1,27 +1,26 @@
 <?php
 /**
  * @license   http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
- * @copyright Copyright (c) 2014 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright Copyright (c) 2014-2016 Zend Technologies USA Inc. (http://www.zend.com)
  */
 
 namespace ZF\ContentValidation;
 
-use Zend\ServiceManager\FactoryInterface;
-use Zend\ServiceManager\ServiceLocatorInterface;
+use Interop\Container\ContainerInterface;
 
-class ContentValidationListenerFactory implements FactoryInterface
+class ContentValidationListenerFactory
```

Comment:
This is a BC break for anybody who might have been extending the factory previously; this is why I've been updating the factories to implement both `__invoke()` and `createService()` signatures. (Doing so is *required* if you want to accept options during invocation for v2 as well.)